website/www/site/content/en/contribute/_index.md

```diff
@@ -57,46 +57,98 @@ There are lots of opportunities to contribute. You can for example:
  https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
  - organize local meetups of users or contributors to Apache Beam
 
+Below is a tutorial for contributing code to Beam, covering our tools and typical process in
+detail.
+
 </div>
 
-<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
-   {{< figure src="/images/arrow-icon_list.svg">}}
+## Get Started
 
-## Contributing code
+### Connect with the Beam community
 
-  </a>
+1. Consider [subscribing to the dev@beam.apache.org](/community/contact-us/) mailing list, especially if you plan to make more than one change or the change will be large. All decisions are consensus-based and happen on the public mailing list.
```

Comment:
Will need to change the link to https://beam.apache.org/community/contact-us/

Reply:
done